src/components/Table/index.tsx

```diff
@@ -103,10 +109,8 @@ const BaseTable = styled.div<{ hasCheckbox: boolean }>`
 
 			&:hover {
 				text-decoration: none;
-				${(props: any) =>
-					!!props.onRowClick || !!props.getRowHref || !!props.onCheck
-						? highlightStyle
-						: ''};
+				${props =>
+					props.hasRowClick || props.hasGetRowRef ? highlightStyle : ''};
```

Comment:
You are missing the `hasCheckbox` check here, I think that's why the visual tests are failing.

Reply:
not sure how i missed that...  (thank you visual tests 😊)